src/core/Events.js

```diff
@@ -96,30 +96,22 @@ L.Evented = L.Class.extend({
 		var typeListeners = this._events[type];
 		if (!typeListeners) {
 			typeListeners = {
-				listeners: {},
+				listeners: [],
 				count: 0

```

Comment:
@fab1an you're correct! We did away with `typeListeners` yesterday! https://github.com/Leaflet/Leaflet/commit/017d29c4ebfaee201b4df48b53ca14e037308aa8